data-access-analysis-tools/index-portal-intro.rst

```diff
@@ -32,34 +32,36 @@ Log in to the Portal Aspect by clicking on the "Portal" panel of the main landin
 The Portal's User Interface
 ===========================
 
-Across the top is a menu bar of interface options, but the DP0 data set is only accessed via the **LSST TAP** service.
+Across the top is a menu bar of interface options, but the DP0 data set is only accessed via the :ref:`Data-Access-Analysis-Tools-TAP`.
 Under **TAP Searches** there are four options.
 
-**1. TAP Service**: Leave this as the default to access DP0 data.
+**1. TAP Service**: Leave this as the default `https://data.lsst.cloud/api/tap` to access DP0 data.
 
-**2. Select Query Type**: Option to query via the single-table interface (default) or ADQL.
 
+.. _single-table-queries:
 
 Single Table Queries
 ====================
 
+**2. Select Query Type**: Select 'Single Table' to query via the single-table interface (default).
+
 **3. Select Table**: Drop down menus of available tables.
 For DP0 data, choose the "Schema: dp01_dc2_catalogs", and then choose the table to query (see the :ref:`DP0-1-Data-Products-DPDD` for a reminder of what tables are available).
 The table view at lower right will automatically update to match the selected table.
 
 **4. Select Contraints**: Only Spatial constraints apply for DP0.1.
-The longitude and latitude columns will automatically update to be the correct column names for right ascencion and declination for the selected table (usually `ra` and `dec` or `coord_ra` and `coord_dec`).
+The longitude and latitude columns will automatically update to be the correct column names for right ascension and declination for the selected table (usually `ra` and `dec` or `coord_ra` and `coord_dec`).
 If a non-existent column name is entered the box will highlight red in indication of the error.
 Choose the desired search method, `Cone` or `Polygon`, and the appropriate instructions for the search terms will appear.
 Keeping the search area to a minimum will keep processing times short and returned subsets small and manageable.
 
-*Note: The examples under the box for coordinates are object names are examples of the formatting only. Those examples are not guaranteed to be in the accessible data sets. The central coordinates for DC2, in decimal degrees, are: 61.863 -35.790. (See Table 2 of `The LSST DESC DC2 Simulated Sky Survey <https://ui.adsabs.harvard.edu/abs/2021ApJS..253...31L/abstract>`_.)*
+*Note: The examples under the box for coordinates are object names as examples of the formatting only. Those examples are not guaranteed to be in the accessible data sets. The central coordinates for DC2, in decimal degrees, are: 61.863 -35.790. (See Table 2 of `The LSST DESC DC2 Simulated Sky Survey <https://ui.adsabs.harvard.edu/abs/2021ApJS..253...31L/abstract>`_.)*
```

Comment:
The link to the DC2 paper isn't rendering correctly at https://dp0-1.lsst.io/v/PREOPS-528/data-access-analysis-tools/index-portal-intro.html

Reply:
I'm going to leave this to Melissa or Matt to look at. I inherited this from the main branch.